tests/framework/checks/skips.go

```diff
@@ -225,3 +225,10 @@ func SkipIfARM64(arch string, message string) {
 		ginkgo.Skip("Skip test on arm64: " + message)
 	}
 }
+
+func SkipIfMultusV3() {
+	const kubevirtWithMultusV3EnvVarKey = "KUBEVIRT_WITH_MULTUS_V3"
+	if os.Getenv(kubevirtWithMultusV3EnvVarKey) == "true" {
```

Comment:
This makes no sense to me.
The way to communicate information from the one that executes the test suite and the test itself is through ginkgo labels, especially if this is about which tests to run.

So I am not clear why we need two mechanisms to do so.

Reply:
I tried to avoid adding more to automation/test.sh, but there is an effort to eliminate skips from inside the test suite (like this one) and use a label filter instead.

I have pushed new changes to remove this skip and use the filter label instead, please take another look.